### ISSUE TITLE
Minidox case 3D printable STL is way too small

### ISSUE THREAD

**Issue by clee** [dotdash32/Cases#2]
[This file](https://github.com/dotdash32/Cases/blob/master/Minidox/3D%20printable%20Case.stl) specifically.

I had to scale it up by 25.4x to make it usable.

The bottom parts were the correct size in my slicer.

**Comment by BadgerOps** [comment]
@clee what did you do for the right side? I'm only seeing that STL for the left side - or did you just mirror it in the slicer?

**Comment by clee** [comment]
@BadgerOps Yup, mirrored in the slicer. I'm guessing the units for this one were set to "inches" when it was saved, instead of millimeters. That'd explain why 25.4x scaling makes it the right size.

**Comment by BadgerOps** [comment]
Ah, yeah that'd make sense! Thanks for the quick response :)
